src/installer/installer.go

```diff
@@ -111,12 +111,8 @@ func (i *installer) InstallNode() error {
 
 	i.UpdateHostInstallProgress(models.HostStageWritingImageToDisk, "")
 
-	image, _ := utils.GetRhcosImageByOpenshiftVersion(i.OpenshiftVersion)
```

Comment:
'GetRhcosImageByOpenshiftVersion' is not used anymore.
Should we delete it?

Reply:
Yup.